tests/bugs/vk-structured-buffer-binding.hlsl.expected

```diff
@@ -0,0 +1,31 @@
+result code = 0
```

Comment:
This test output could end up being brittle, because even whitespace changes in our GLSL output would lead to a failure. It is for that reason that the other tests for cross-compilation (like the ray tracing test you also had to modify) don't compare Slang's GLSL output against a baseline, but instead compare the SPIR-V output we get from both the Slang file and a comparable GLSL file. That should guard against getting spurious test failures when we make minor changes to formatting, etc.

I'm fine with this test as written for now, but if/when we have to revise it based on a change in the code emit logic we should go ahead and replace it with the other kind of test.